src/group_theory/exponent.lean

```diff
@@ -52,19 +53,40 @@ variables (G) [monoid G]
   for all `g`.-/
 @[to_additive "A predicate on an additive monoid saying that there is a positive integer `n` such
   that `n • g = 0` for all `g`."]
-def exponent_exists  := ∃ n, 0 < n ∧ ∀ g : G, g ^ n = 1
+def is_torsion := ∃ n, 0 < n ∧ ∀ g : G, g ^ n = 1
```

Comment:
A torsion group should be one where every element is torsion, which isn't the same as the exponent existing. For instance the direct sum of cyclic groups of all possible orders is torsion but the exponent doesnt exist.

Reply:
Thanks for correcting my mistake Alex, and for the example. I'm working on defining these separately and leaving what's here as-is then.